### PR TITLE
Automatically convert TRUSTXXX -> TRUST

### PR DESCRIPTION
**Pull request by wilzbach**
```bash
replacements=(
"TRUSTdefault default_"
"TRUSTsystem system"
"TRUSTtrusted trusted"
"TRUSTsafe safe"
)
shopt -s globstar
for r in "${replacements[@]}" ; do
    w=($r)
    sed "s/${w[0]}/TRUST.${w[1]}/g" -i **/*.d
done
```

### CODE REVIEW

**Review by PetarKirov — APPROVED** [review]

**Review by PetarKirov — APPROVED** [review]

File: src/dmd/mtype.h [hunk 1]
```diff
@@ -549,10 +549,10 @@ enum RET
...
 enum TRUST
 {
-    TRUSTdefault = 0,
-    TRUSTsystem = 1,    // @system (same as TRUSTdefault)
-    TRUSTtrusted = 2,   // @trusted
-    TRUSTsafe = 3,      // @safe
+    default_ = 0,
+    system = 1,    // @system (same as TRUST.default)
+    trusted = 2,   // @trusted
+    safe = 3,      // @safe
```
Comment (ZombineDev) [inline src/dmd/mtype.h]: Fails with:
```
dmd/mtype.h:553: error: 'system' redeclared as different kind of symbol
/usr/include/stdlib.h:113: error: previous declaration of 'int system(const char*)'
```

Maybe suffix all enum members with an underscore?

Comment (wilzbach) [reply src/dmd/mtype.h]: The problem was that enum doesn't create its own scope in C++ 98.

**Review by ibuclaw — DISMISSED** [review]
Don't bother updating the headers, it's doing no one any favours. Recent changes should be reverted.

**Comment by wilzbach** [comment]
> Don't bother updating the headers, it's doing no one any favours. Recent changes should be reverted.

Cool! Done.

**Pull request merged**